### PR TITLE
Added react-router-dom and routing

### PR DESCRIPTION
**Pull request by mvkv**
created components for views,
formatted code

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,67 +1,78 @@
...
 
-import './Navbar.css'
+import "./Navbar.css";
 
 const MenuItems = [
   {
     title: "About",
-    target: "#About",
-    cName: "nav-link"
+    target: "About",
```
Comment (sspacelynx) [inline src/components/Navbar.js:10]:
```suggestion
    target: "/About",
```
Also do this for every other target in this array.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+import Events from "./views/Events";
+import PassionInAction from "./views/PassionInAction";
+import Join from "./views/Join";
+import Hosts from "./views/Hosts";
 
 function Site() {
   return (
-    <Navbar />
-  )
+    <Router>
```
Comment (sspacelynx) [inline src/index.js:15]: To make react router work on subdirectories (for dev deployment), we'll have to set `basename` argument to the full subdir path, but I'll take care of that in a following PR.

**Review by sspacelynx — CHANGES_REQUESTED** [review]
Minor code consistencies nitpicks. The `Link` class is doing weird stuff when serving the website from a subdir, as it works correctly for navbar links but not for the `Join Us` button or for the logo for some reason. Will try to fix that in a following PR.
Otherwise LGTM.

File: src/components/Navbar.js [hunk 3]
```diff
@@ -1,67 +1,78 @@
...
-  {
-    title: "News",
-    target: "#News",
+    target: "PassionInAction",
     cName: "nav-link",
   },
-]
+];
 
 class Navbar extends React.Component {
```
[the remark this answers is not in the document]
Comment (mvkv) [reply src/components/Navbar.js:30]: Done

File: src/components/Navbar.js [hunk 4]
```diff
@@ -1,67 +1,78 @@
-import React from 'react'
-import logo_white from '../logo_white.svg'
+import React from "react";
```
[the remark this answers is not in the document]
Comment (mvkv) [reply src/components/Navbar.js:1]: Done

File: src/components/Navbar.js (hunk 1 above)
Comment (mvkv) [reply src/components/Navbar.js:10]: Done

File: src/components/Navbar.js [hunk 5]
```diff
@@ -1,67 +1,78 @@
...
+        </div>
       </nav>
-    )
+    );
   }
 }
 
-export default Navbar
\ No newline at end of file
+export default Navbar;
```
[the remark this answers is not in the document]
Comment (mvkv) [reply src/components/Navbar.js:78]: Done

File: src/index.js (hunk 2 above)
Comment (mvkv) [reply src/index.js:15]: Done

**Pull request merged**